### PR TITLE
Update pilot widget

### PR DESCRIPTION
**Pull request by LukeSeers**

### CODE REVIEW

File: modules/portmaster/src/app/widgets/pilot-widget/pilot-widget.html [hunk 1]
```diff
@@ -1,42 +1,32 @@
...
+    <path class="shield-ok" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.8"
+      d="M8.712 12.566l2.193 2.193 4.787-4.788" /> 
+    <path class="shield-warn" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.8" 
+      d="M12 11V7.75M12 14.63a.16.16 0 10.16.16.16.16 0 00-.16-.16" transform="translate(-11.09 -7)"/>
+    <path class="shield-fail" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.6" 
+    d="M14.83 9.17l-5.66 5.66M14.83 14.83L9.17 9.17"/>
   </svg>
-
   <h2>{{lockLevel?.displayText}}</h2>
+  <span>Running smoothly</span>
```
Comment (dhaavi) [inline modules/portmaster/src/app/widgets/pilot-widget/pilot-widget.html:17]: Maybe not.

**Review by dhaavi — CHANGES_REQUESTED** [review]
I tested it an it works. 👌

I'm not quite sure why we got started with `Failure`, let's talk about that:
https://github.com/safing/portmaster-ui/blob/a3af21d55eec13cab79c85880809795afe23e591/modules/portmaster/src/app/widgets/pilot-widget/pilot-widget.ts#L114

**Review by dhaavi — APPROVED** [review]

**Comment by ppacher** [comment]
The animation of the lock has a little start delay which causes the lock to look weired when first rendered. This might not be a real issue but if you start to drag the pilot widget around you "re-create" the lock at a different DOM position which will reset the animation to start. That looks a bit weird:

animation-issue

**Review by ppacher — CHANGES_REQUESTED** [review]
See other comment

**Review by dhaavi — APPROVED** [review]

**Review by ppacher — COMMENTED** [review]
LGTM

**Pull request merged**